### ISSUE TITLE
ListView doesn't show a scrollbar when assigned to a HPaned.Panel2.Content

### ISSUE THREAD

**Issue by steffenWi** [mono/xwt#491]
Hi,

so I've a ListView I'd like to display inside a HPaned widget. The ListView is displayed but if I resize the window, no scrollbars are displayed on the ListView, instead the ListView is 'cut off' where it 'hits' the window.

As far as I can tell, the ListView is not resized when the parent's width is reduced. I also tried placing it inside a VBox since that is, essentially, what the sample is (successfully) doing, but it didn' work either.

I also am not able to find any Event that could help me (Resize for example) here. So...how is this supposed to work?

**Comment by sevoku** [comment]
Hhm, a list should not be packed into a ScrollView and scroll on its own. Exactly like in the ListView examples. Could you gist some example, where it doesn't work?

To handle size changes you could subscribe the lists BoundsChanged event.

**Comment by steffenWi** [comment]
erhh, sorry for the wait. Well, the ListView, in my case, is not inside a ScrollView. It's the content of a HPaned.Panel. I set it to Fill but, as I said, there are no scrollbars. I tried putting it inside a ScrollView but that did not work either. What I also do not really understand is that the Size.Height/Width properties of the ListView always report the size of the visible part of the ListView, no it's entire size.

I'll try to use BoundsChanged event for now, that should help/work.
